e2e-tests/cypress/integration/common/guidance-pages.js

```diff
@@ -0,0 +1,17 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+When('the appellant navigates to the next page', () => {
+  cy.guidancePageNavigation('next');
+});
+
+When('the appellant navigates to the previous page', () => {
```

Comment:
Why have we added this in current step definition

Reply:
We know it is needed generically for all so made sense to add it now

Reply:
But we shouldn't write any extra code even though we know that it will be used later.